src/sparql/qc/mondo/qc-gene-identifier-mismatch.sparql

```diff
@@ -0,0 +1,35 @@
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+
+# Find classes with mismatched gene identifiers added as equivalentTo and subClassOf
+
+SELECT DISTINCT ?entity ?label ?equivGeneIdentifier ?subClassGeneIdentifier
+WHERE {
+  ?entity rdf:type owl:Class ;
+          rdfs:label ?label .
+  
+  # Equivalent class restriction
+  ?entity owl:equivalentClass ?equivClass .
+  ?equivClass owl:intersectionOf/rdf:rest*/rdf:first ?equivComponent .
+  
+  ?equivComponent rdf:type owl:Restriction ;
+                  owl:onProperty obo:RO_0004003 ;
+                  owl:someValuesFrom ?equivGeneIdentifier .
+  
+  # subClassOf restriction
+  ?entity rdfs:subClassOf ?subClassRestriction .
+  ?subClassRestriction rdf:type owl:Restriction ;
+                       owl:onProperty obo:RO_0004003 ;
+                       owl:someValuesFrom ?subClassGeneIdentifier .
+
+  # Filter for gene identifiers with HGNC or NCBIGene prefixes
+  FILTER(STRSTARTS(STR(?equivGeneIdentifier), "http://identifiers.org/hgnc/") || 
+         STRSTARTS(STR(?equivGeneIdentifier), "http://identifiers.org/ncbigene/"))
```

Comment:
I assume you checked that they are `http` vs `https`.

Reply:
Thanks for asking. Yes, I checked it is `http`. The terms to import now come in through this [gSheet](https://docs.google.com/spreadsheets/d/1UME3pTeR42hwNt1I6RPl0nvJKVfTIeJqhx80laAlD50/edit?gid=0#gid=0) that we pull from to add into `manual_seed.txt` when refreshing imports and there are also already qc failures of this check that we are resolving in another [ticket](https://github.com/monarch-initiative/mondo/issues/8329).